pallets/loans/src/lib.rs

```diff
@@ -63,7 +65,7 @@ pub mod module {
     use super::*;
 
     #[pallet::config]
-    pub trait Config: frame_system::Config {
+    pub trait Config: frame_system::Config + pallet_prices::Config {
```

Comment:
Could you talk more about this?

Reply:
@yz89 

We were getting `OracleCurrencyPriceNotReady ` because of https://github.com/parallel-finance/parallel/blob/defect/fix-oracle-price-benchmarking/pallets/loans/src/loan.rs#L134-L135

To fix that, I had to add `EmergencyPrice`,  https://github.com/parallel-finance/parallel/blob/defect/fix-oracle-price-benchmarking/pallets/loans/src/benchmarking.rs#L38

Thats why I added `pallet_prices`

Reply:
Emergency price is a different logic, probably not in good shape.
We better not use trait inheritance, it makes code hard to test and less structural. 
The error `OracleCurrencyPriceNotReady` should be resolved if you have a mock PriceFeeder.

Reply:
I just checked. Even if we import orml_oracle, lot of changes are required in loan pallet. I am trying to find a soltion.